typescript/src/world-company-remuneration/basket-informations.ts

```diff
@@ -0,0 +1,46 @@
+const K = "K";
+
+/**
+ * Created by thomas on 02/12/2019.
```

Comment:
You don't need to mention your name and date - tools like gitlens should be enough.

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway